src/domain/session/room/RoomViewModel.js

```diff
@@ -285,6 +289,13 @@ export class RoomViewModel extends ViewModel {
             console.error(err.stack);
         }
     }
+
+    toggleDetailsPanel() {
+        const isPanelOpen = !!this.navigation.path.get("details");
+        const link = isPanelOpen ? this.urlCreator.urlUntilSegment("room"):
+        this.urlCreator.urlForSegment("details");
+        window.location.href = link;
```

Comment:
view models should not access any DOM interfaces, like window, as they are assume to work outside of browser environments. The easiest way would probably to expose a url property from this view model.